api/USERS_GUIDE.md

```diff
@@ -0,0 +1,154 @@
+# API User's Guide
+
+The DP code provides a RESTful API for accessing various its of data
```

Comment:
"its" -> "items"?

Reply:
Actually, "bits". Thanks, I've pushed a small update to master to fix this.